authenticator/filetokencache_test.go

```diff
@@ -9,16 +9,18 @@ import (
 )
 
 func TestFileTokenCache_New(t *testing.T) {
-	defer os.Remove("/tmp/gotransip_test123")
+	tmpFile := os.TempDir() + "/gotransip_test123"
```

Comment:
Made this PR on Windows so /tmp doesn't work. We can either use `os.TempDir` or use a relative path.

Reply:
You could even just use [ioutil.TempFile](https://golang.org/pkg/io/ioutil/#TempFile) to make it even more transparent! Perhaps it goes beyond the scope of this MR but that should work on any OS

Reply:
ioutil.TempFile actually creates on `*os.File`, so that won't work here. I've changed the code to use filepath.Join() in combination with os.TempDir so we use the correct path separator.